mllib/src/main/scala/org/apache/spark/ml/clustering/KMeans.scala

```diff
@@ -258,6 +290,27 @@ class KMeans @Since("1.5.0") (
   @Since("1.5.0")
   def setSeed(value: Long): this.type = set(seed, value)
 
+  /** @group setParam */
+  @Since("2.0.0")
+  def setInitialModel(value: KMeansModel): this.type = set(initialModel, value)
```

Comment:
We can leave the discussion for another JIRA. Has it been created? If not, we should create it before this is merged.